### ISSUE TITLE
Consensus-layer Call 130

### ISSUE THREAD

**Issue by djrtwo** [ethereum/pm#987]
# Consensus-layer Call 130

[prev: call 129](https://github.com/ethereum/pm/issues/973)

Meeting Date/Time: [Thursday 2024/3/21 at 14:00 UTC](https://savvytime.com/converter/utc/mar-21-2024/2pm)  
Meeting Duration: 1.5 hours
[stream](https://youtube.com/live/3rRJ1H0MJDY?feature=share)

1. Deneb
    * [Move Dencun EIPs to final](https://github.com/ethereum/pm/issues/987#issuecomment-2007457580)
2. [local block boost defaults](https://github.com/ethereum/pm/issues/987#issuecomment-2010459595)
3. Electra
    * [ILs update](https://github.com/ethereum/pm/issues/987#issuecomment-2009970451)
    * [maxEB discussion](https://github.com/ethereum/pm/issues/987#issuecomment-2010332963)
    * [Time-based blob gas increase](https://github.com/ethereum/pm/issues/987#issuecomment-2011044933)
    * https://github.com/ethereum/EIPs/pull/8340
    * https://github.com/ethereum/EIPs/pull/8341
5. Research, spec, etc
    * https://github.com/ethereum/consensus-specs/pull/3623
    * https://github.com/libp2p/specs/pull/548
    * Mplex still needed?
7. Open Discussion/Closing Remarks

**Comment by timbeiko** [comment]
Reminder to move Dencun EIPs to Final: https://github.com/ethereum/EIPs/pull/8334

**Comment by AgeManning** [comment]
Some misc network-related points to raise (non of which require a hard fork :+1:) :

- Node-id to shard mapping: https://github.com/ethereum/consensus-specs/pull/3623
- IDONTWANT in gossipsub: https://github.com/libp2p/specs/pull/548
- Mplex - Do we still want/need it? @jxs

**Comment by michaelneuder** [comment]
IL updates: 

- breakout room 
  - recording: https://www.youtube.com/watch?v=GVdkDXZTtnw
  - notes https://hackmd.io/@ttsao/Bkg53AHAT
  - takeaway: "we agree on the PoC features" 
- Potuz's doc on bundling https://hackmd.io/@potuz/Sk_Xg0x06
- Terence's doc on blobs (not in PoC) https://hackmd.io/@ttsao/ryzAw9PRa
- The case for ILECTRA https://notes.ethereum.org/@mikeneuder/the-case-for-ilectra
- PoC spec https://notes.ethereum.org/@mikeneuder/il-poc-spec

**Comment by philknows** [comment]
Here are some notes relating to maxEB from today's session:
https://hackmd.io/@philknows/BkEL0Fu0p

There is also another one scheduled for Friday through @ethDreamer

**Comment by fredriksvantes** [comment]
Proposal to set default local block boost to 10%.
Related PRs in [Prysm](https://github.com/prysmaticlabs/prysm/pull/13772), [Lodestar](https://github.com/ChainSafe/lodestar/pull/6568), [Lighthouse](https://github.com/sigp/lighthouse/pull/5441), [Nimbus](https://github.com/status-im/nimbus-eth2/pull/6103) and [Teku](https://github.com/Consensys/teku/pull/8108)
 
Testing team planning to implement tests for this in [Hive](https://github.com/ethereum/hive/issues/1062)

**Comment by adietrichs** [comment]
With [EIP-7623](https://eips.ethereum.org/EIPS/eip-7623) aiming to increase calldata cost and that way reduce the worst-case block size, and the time between now and Pectra allowing us to gain more experience with 4844 blob throughput levels, we have an EIP draft to gradually increase blob throughput. I will have a draft EIP PR to share tomorrow closer to the call, but in the meantime this graph by @michaelneuder illustrates the idea very well:
![image](https://github.com/ethereum/pm/assets/10457348/b789d282-2f31-4f61-9302-19281391a246)
Basically, over a time of 4 months after Pectra, gradually increase throughput from today's 3/6 to a final 8/16.
I'd like to briefly present that draft EIP tomorrow.

**Comment by potuz** [comment]
Please make sure to talk about account abstraction in the context of inclusion lists. The current design imposes a restriction to account abstraction in that an account cannot be drained by another one

**Comment by etan-status** [comment]
- https://github.com/ethereum/EIPs/pull/8340
- https://github.com/ethereum/EIPs/pull/8341

**Comment by adietrichs** [comment]
> I will have a draft EIP PR to share tomorrow closer to the call

Here is the link to the draft PR for the blob throughput increase: https://github.com/ethereum/EIPs/pull/8343

**Comment by tvanepps** [comment]
[Dencun Diary](https://stateful.mirror.xyz/x7uPWur2ELBP6jkKEILKhAwPCJvCv10s3UrS1y1qpcc) was published yesterday

> +45 core Ethereum contributors looking back on the last two years, and looking ahead to what’s next

**Comment by timbeiko** [comment]
closed in favor of https://github.com/ethereum/pm/issues/1003
